packages/pytest-simcore/src/pytest_simcore/helpers/playwright_sim4life.py

```diff
@@ -5,9 +5,8 @@
 from typing import Final, TypedDict
 
 import arrow
+from common_library.pydantic_type_adapters import ByteSizeAdapter
```

Comment:
be careful here. I'm not sure you are not adding that dependency in the playwright stuff with this..

Reply:
let's revert this one